### PR TITLE
Viewing Options, and bug fixing

### PR DESCRIPTION
**Pull request by okbel**
# Viewing Options and Bug Fixing

## What does this PR do?
- Adds Viewing Options to the Moderation View
- Fix Header Full-Width
- Fix Table View - Icon Drawer
- FIx Drawer
- Adding container to the Moderation View
	- Aligned Moderation Keys under container
	- Aligned View Options

The Dropdown Component is too unstable. I will introduce a new Dropdown component in the next PRs.

### CODE REVIEW

**Comment by kgardnr** [comment]
This is so great @okbel!!!! 💯

File: client/coral-admin/src/components/ui/Drawer.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ import styles from './Drawer.css';
 import t from 'coral-framework/services/i18n';
 import {can} from 'coral-framework/services/perms';
 
-const CoralDrawer = ({handleLogout, auth}) => (
-  <Drawer className={styles.header}>
+const CoralDrawer = ({handleLogout, auth = {}}) => (
```
Comment (alinhle) [inline client/coral-admin/src/components/ui/Drawer.js:9]: Just a note that dont need to be addressed in this PR. Why is `Drawer` in the folder `ui`. I thought ui is only for "generic ui building blocks".

**Review by alinhle — COMMENTED** [review]
image

I think the viewing options should never be _behind_ the comments but should trigger the other layout earlier, so that it does not happen.

**Comment by okbel** [comment]
That screenshot looks from an unreacheable resolution. I will lower the breakpoint.

**Comment by alinhle** [comment]
What's an unreachable resolution?

File: client/coral-admin/src/components/ui/Drawer.js (hunk 1 above)
Comment (okbel) [reply client/coral-admin/src/components/ui/Drawer.js:9]: Yeah, that one should be definitelly out.

**Comment by okbel** [comment]
Ready to be tested! Thank you :) @cvle

**Review by alinhle — APPROVED** [review]

**Pull request merged**